src/state/user/reducer.ts

```diff
@@ -99,6 +102,7 @@ export const initialState: UserState = {
   timestamp: currentTimestamp(),
   URLWarningVisible: true,
   userAdvancedSwapDetails: true,
+  closedStacklyPopup: false,
```

Comment:
Optional:
Instead of using `closedStacklyPopup` suggest using `stacklyPopupOpen`